### DISCUSSION TITLE
NodeJS Console App - Inspect

### DISCUSSION THREAD

**Discussion by TomFuch** [statelyai/xstate#3595] [Q&A] [answered]
Hi...
Is it possible to inspect a Console App without having a window Object?

Thanks for Hints...

**Comment by Andarist** [comment, marked as answer]
There is an example of how to inspect your node app using WebSockets: https://github.com/statelyai/xstate/blob/9e35294e9c13172c4822b4849d16c88189e33c2a/packages/xstate-inspect/examples/server.ts

**Reply by TomFuch** [reply]
Great - Thank you!!!
